### PR TITLE
🐛 FMS real/show fix totals under dashboard + Buildinghover fix cellBuildings

### PR DESCRIPTION
**Pull request by JRH-1997**
I hope this will Fix the issue that the status sum is fms_real on the dashboards. 
Could you look if it's affecting an other module?

### CODE REVIEW

**Review by KBOE2 — APPROVED** [review]

**Review by Piet2001 — APPROVED** [review]

**Comment by JRH-1997** [comment]
Add buildingHover fix, buildingType '6' of cellbuilding was hardcoded.

File: src/modules/buildingHover/main.ts [hunk 1]
```diff
@@ -67,7 +67,11 @@ export default (async (LSSM, MODULE_ID) => {
                 }&nbsp;<i class="fa fa-users"></i>&nbsp;${
                     building.personal_count
                 }`;
-                if (building.building_type === 6)
+                if (
+                    Object.values(LSSM.$t('cellBuildings')).indexOf(
+                        building.building_type
+                    ) >= 0
+                )
```
Comment (KBOE2) [inline src/modules/buildingHover/main.ts:74]: This does work but I would prefer using newer Code with .includes function:
```suggestion
if(Object.values(LSSM.$t('cellBuildings')).includes(building.building_type))
```

**Review by KBOE2 — CHANGES_REQUESTED** [review]

**Comment by JRH-1997** [comment]
I used the same code that was used for other buildings in that file. I would also prefer this code.

**Comment by KBOE2** [comment]
Oh, looks like I haven't used include myself 😅
If you want to, you can also change the others 🙃

**Review by KBOE2 — APPROVED** [review]

**Comment by JRH-1997** [comment]
I will have a look that when I'm at my pc again.

**Review by KBOE2 — APPROVED** [review]

**Review by Suchty112 — APPROVED** [review]

**Pull request merged**